client/app/components/land-use-action.js

```diff
@@ -0,0 +1,100 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class LandUseActionComponent extends Component {
+  // represents pasForm model
+  @tracked pasForm;
+
+  // local variable that records results of dropdown selection
+  selectedLandUseAction = {};
```

Comment:
Good distinction between local variable and an argument variable... 

How come we need to store state on this component? Is there some reason the model alone can't be responsible for that?